codecov.yml

```diff
@@ -8,3 +8,23 @@ coverage:
       default:
         target: 80%
         threshold: 1%  # the leniency in hitting the target
+
+comment:
```

Comment:
Added this, but it seems a bit messed up. I don't think the coverage is universally 100% in those areas

![Screenshot from 2024-10-01 11-27-01](https://github.com/user-attachments/assets/bad513f4-c4d9-4499-b000-bf708234fb41)

Reply:
Nvm. this seems to be patch-related coverage. It now looks different/ok.

Reply:
If it works, we could include running the tests for the other 2 test/coverage workflows.